docs/issues.md

```diff
@@ -4,11 +4,11 @@ By following this guide, you will have a fully functional Shopify app. However,
 
 ## Notes on session handling
 
-Before you start writing your application, please note that the Shopify library stores some information for OAuth in sessions. Since each application may choose a different strategy to store information, the library cannot dictate any specific storage strategy. By default, `Shopify.Context` is initialized with `SQLiteSessionStorage`, which will enable you to start developing your app by storing sessions using the file-based SQLite package. You can quickly get your app set up using it, but please keep the following in mind.
+Before you start writing your application, please note that the Shopify library stores some information for OAuth in sessions. Since each application may choose a different strategy to store information, the library cannot dictate any specific storage strategy. By default, `config` is initialized with `SQLiteSessionStorage`, which will enable you to start developing your app by storing sessions using the file-based SQLite package. You can quickly get your app set up using it, but please keep the following in mind.
```

Comment:
For now, this is temporarily using `Memory`, but we might want to make it configurable per runtime adapter, since there will likely not be one one-size-fits-all solution.

Reply:
Setting up a separate PR to tackle that as this one is too big with all the search-replace going on.